src/Extensions/Hosting.Services.Web/HostBuilderExtensions.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Fabric;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.ServiceFabric.Services.Communication.AspNetCore;
+
+namespace Microsoft.Omex.Extensions.Hosting.Services.Web
+{
+	/// <summary>
+	/// Extension to add Omex dependencies to HostBuilder
+	/// </summary>
+	public static class HostBuilderExtensions
+	{
+		/// <summary>
+		/// Add Kestrel service listener to stateless service
+		/// </summary>
+		public static ServiceFabricHostBuilder<StatelessServiceContext> AddKestrelListener<TStartup>(
+			this ServiceFabricHostBuilder<StatelessServiceContext> builder,
+			string name,
+			ServiceFabricIntegrationOptions options,
+			Action<IWebHostBuilder>? builderExtension = null)
+				where TStartup : class =>
+			builder.AddKestrelListener<TStartup, StatelessServiceContext>(name, options, builderExtension);
+
+
+		/// <summary>
+		/// Add Kestrel service listener to stateful service
+		/// </summary>
+		public static ServiceFabricHostBuilder<StatefulServiceContext> AddKestrelListener<TStartup>(
+			this ServiceFabricHostBuilder<StatefulServiceContext> builder,
```

Comment:
You can just expose the private method rather than these two public methods. The effect will be the same and the callers should just work as is

Reply:
The reasoning behind having two private methods described here https://github.com/microsoft/Omex/pull/153#discussion_r389410264